tests/data/doc_parse/spdx-expected.json

```diff
@@ -34,6 +34,7 @@
   ],
```

Comment:
You should test with the latest [SPDX example files](https://github.com/spdx/spdx-spec/tree/development/v2.3/examples)

Reply:
Thank you so much @goneall! 

Checking out the referenced example files